orthanc/orthanc-anon/plugin/pixl.py

```diff
@@ -132,6 +133,22 @@ def AzureDICOMTokenRefresh():
     return None
 
 
+def Send(resourceId: str) -> None:
+    """Send the resource to the appropriate destination"""
+    msg = f"Sending {resourceId}"
+    logger.debug(msg)
+
+    slug = get_project_slug_from_hashid(_get_patient_id(resourceId))
```

Comment:
nested call is interesting, Probably worth having them as a separate variable for readability?